content/browser/web_contents/web_contents_impl.cc

```diff
@@ -9314,6 +9322,9 @@ WebContentsImpl* WebContentsImpl::GetResponsibleWebContents() {
 }
 
 WebContentsImpl* WebContentsImpl::GetFocusedWebContents() {
+  if (!GetFocusedFrameTree()) {
+    return nullptr;
```

Comment:
Don't know whether we should update caller in content\browser\find_request_manager.cc to handle the possible nullptr return value from GetFocusedWebContents()

Reply:
find-in-page indeeds needs hardening against null focused frame tree. I see a crash without it. Updated.